### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kirylpl**
Adding terminal command for updating gitignore

### CODE REVIEW

**Comment by bf4** [comment]
@kirylpl Do you know if that works for everyone, including windows users?

**Comment by kirylpl** [comment]
@bf4 yes, echo is cross-platform command https://en.wikipedia.org/wiki/Echo_(command)

**Comment by bf4** [comment]
And >> ?

B mobile phone

**Comment by kirylpl** [comment]
@bf4 You were right. There is no need in comma on windows. Updated.

File: README.md [hunk 1]
```diff
@@ -87,7 +87,11 @@ Getting started
    are not tracked by Git (optional):
 
     ```
-    coverage
+    echo "coverage" >> .gitignore
+    ```
+    Or if you use Windows for some reason:
```
Comment (PragTob) [inline README.md]: I think we should get rid of the `for some reason` part - no need to belittle Windows users.

Comment (kirylpl) [reply README.md]: @PragTob Proudly speaking, never met ruby developer using Windows. So, imo, it will not touch anybody.

Comment (PragTob) [reply README.md]: I know quite some of them also including popular Rubyists :)

Comment (kirylpl) [reply README.md]: @PragTob Got it, I add some row for this fanatics also :)

File: README.md [hunk 2]
```diff
@@ -87,7 +87,11 @@ Getting started
    are not tracked by Git (optional):
 
     ```
-    coverage
+    echo "coverage" >> .gitignore
+    ```
+    Or if you use Windows:
+    ```
+    echo coverage >> .gitignore
```
Comment (bf4) [inline README.md]: am I missing something or is the only difference the quotes? If so, you can omit quotes on *nix as well. 

My own feeling is that if you can't be trusted to know how to add a line to your gitignore you have bigger issues.

**Pull request merged**